plugins/git/git.plugin.zsh

```diff
@@ -145,7 +145,7 @@ compdef _git gk='gitk'
 alias gke='\gitk --all $(git log -g --pretty=format:%h)'
 compdef _git gke='gitk'
 
-alias gl='git pull'
```

Comment:
I am using `gl` command. plz add a new alias for `git pull` or please consider other solution.